tasks/HelpMeCarry.tex

```diff
@@ -0,0 +1,76 @@
+\section{Help Me Carry [Party Host]}
+The robot helps a guest to carry some luggage to his/her car.
```

Comment:
- ~a guest~ → the operator
- ~his/her car~ → the cab parked outside

Reply:
A personal car parked outside makes more sense because the robot does not know the car's location and it must follow the person. Otherwise, the robot can find the cab outside and go it without following the person.

Reply:
then *the car* would do.